src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -537,18 +537,14 @@ class ReplaceClauseSegment(BaseSegment):
     type = "select_replace_clause"
     match_grammar = Sequence(
         "REPLACE",
-        OneOf(
-            # Multiple replace in brackets
-            Bracketed(
-                Delimited(
-                    # Not *really* a select target element. It behaves exactly
-                    # the same way however.
-                    Ref("SelectClauseElementSegment"),
-                    delimiter=Ref("CommaSegment"),
-                )
-            ),
-            # Single replace not in brackets.
-            Ref("SelectClauseElementSegment"),
+        # Multiple replace in brackets
```

Comment:
Let's remove this comment as it's singular or multiple now

```suggestion
```

Reply:
Yeah, np - pushed an update.